File: mpp-proiect-repository-cristinahendre/lab9(Java)/Java/lab6/modul/NetworkModule/src/main/java/dto/VoluntarDTO.java
package dto;


import java.io.Serializable;


public class VoluntarDTO implements Serializable{
    private String email;
    private String passwd;
    private String nume;
    private String prenume;

    public void setPasswd(String passwd) {
        this.passwd = passwd;
    }

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public String getPrenume() {
        return prenume;
    }

    public  VoluntarDTO(String email ,String pass){
        this.email=email;
        this.passwd=pass;
    }

    public void setPrenume(String prenume) {
        this.prenume = prenume;
    }

    public VoluntarDTO(String id, String passwd, String n, String p) {
        this.email = id;
        this.passwd = passwd;
        this.nume=n;
        this.prenume=p;
    }

    public String getEmail() {
        return email;
    }

    public void setId(String id) {
        this.email = id;
    }

    public String getPasswd() {
        return passwd;
    }

    @Override
    public String toString(){
        return "VoluntarDTO["+email+' '+passwd+"]";
    }
}

File: mpp-proiect-repository-cristinahendre/lab6(web)/CSharp/Lab5/Network/VoluntarDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Network
{
	[Serializable]
	public class VoluntarDTO
	{
		private string email;
		private string passwd;
		private string nume;
		private string prenume;

		

		public VoluntarDTO(string email, string passwd)
		{
			this.email = email;
			this.passwd = passwd;
		}

		public VoluntarDTO(string nume ,string prenume, string email, string passwd)
		{
			this.email = email;
			this.passwd = passwd;
			this.nume = nume;
			this.prenume = prenume;
		}

		public virtual string Email
		{
			get
			{
				return email;
			}
			set
			{
				this.email = value;
			}
		}


		public virtual string Nume
		{
			get
			{
				return nume;
			}
			set
			{
				this.nume = value;
			}
		}

		public virtual string Prenume
		{
			get
			{
				return prenume;
			}
			set
			{
				this.prenume = value;
			}
		}



		public virtual string Passwd
		{
			get
			{
				return passwd;
			}
            set
            {
				this.passwd = value;
            }
		}
	}
}
